drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/radio_test/fmac_api.h

```diff
@@ -99,8 +99,8 @@ enum nrf_wifi_status nrf_wifi_fmac_radio_test_prog_rx(struct nrf_wifi_fmac_dev_c
  * @param lna_gain LNA gain value.
  * @param bb_gain Baseband gain value.
  *
- * This function is used to send a command to:
- *	- The RPU firmware to start the RF test capture in radio test mode.
+ * This function is used to send a command to the RPU firmware to start the
```

Comment:
Please move unrelated changes to a separate commit.

Reply:
Reverted this change.